portality/templates/doaj/article.html

```diff
@@ -87,7 +87,7 @@ <h1>&lt; Article title missing &gt;</h1>
 
         {# doi #}
         {% if doi %}
-            DOI {{ doi | doi_url | safe}}
+            DOI <a href="{{ doi | doi_url | safe }}"> {{ doi }}</a>
```

Comment:
We changed the behaviour below to handle if the DOI is non canonical, and returns an empty string, so we need to do something similar here.

Probably the best thing to do is normalise the DOI once at the top, and then it will be easy just to check for a non empty string wherever you might display the DOI.

Reply:
the doi_url is using normalized doi already as it is changed in the app.py method. So we can only check if the string is not empty and do not display it if so. Question is if we want to display doi as it is or normalized here (everywhere?)